PlasmaAnchor/anchor.md

```diff
@@ -0,0 +1,117 @@
+## Anchor: Finality Gadget on Tendermint
+
+### Motivation
```

Comment:
I think it would be nice to state at the beginning that this need for a validator set > 1